modules/programs/screen.nix

```diff
@@ -0,0 +1,29 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.screen;
+
+in
+{
+  options = {
+    programs.screen = {
+      enable = mkEnableOption "GNU screen";
+
+      settings = mkOption {
+        type = types.string;
+        default = {};
```

Comment:
`{}` is not a valid string value, I believe you want `""` here.

Reply:
Sigh. I'm too used to languages which check such things for me